content/en/agent/guide/heroku-ruby.md

```diff
@@ -225,32 +217,305 @@ Collector
-      Total Runs: 9
-      Metric Samples: Last Run: 15, Total: 135
-      Events: Last Run: 0, Total: 0
-      Service Checks: Last Run: 1, Total: 9
-      Average Execution Time : 102ms
-      Last Execution Date : 2021-05-11 14:14:34 UTC (1620742474000)
-      Last Successful Execution Date : 2021-05-11 14:14:34 UTC (1620742474000)
-      metadata:
-        version.major: 13
-        version.minor: 2
-        version.patch: 0
-        version.raw: 13.2 (Ubuntu 13.2-1.pgdg20.04+1)
-        version.scheme: semver
-
+  postgres (5.4.0)
+  ----------------
+    Instance ID: postgres:e07ef94b907fe733 [OK]
+    Configuration Source: file:/app/.apt/etc/datadog-agent/conf.d/postgres.d/conf.yaml
+    Total Runs: 9
+    Metric Samples: Last Run: 15, Total: 135
+    Events: Last Run: 0, Total: 0
+    Service Checks: Last Run: 1, Total: 9
+    Average Execution Time : 102ms
+    Last Execution Date : 2021-05-11 14:14:34 UTC (1620742474000)
+    Last Successful Execution Date : 2021-05-11 14:14:34 UTC (1620742474000)
+    metadata:
+      version.major: 13
+      version.minor: 2
+      version.patch: 0
+      version.raw: 13.2 (Ubuntu 13.2-1.pgdg20.04+1)
+      version.scheme: semver
 
 [...]
 ```
 
-Once you have checked that the Postgres check is running correctly, you can start looking at the Postgres metrics available on the [Metrics Summary][14] page:
+Once you have checked that the Postgres check is running correctly, you can start looking at the Postgres metrics available on the [Metrics Summary][15] page:
 
 {{< img src="agent/guide/heroku_ruby/postgres_metrics.png" alt="Datadog Metrics Explorer" >}}
 
+### Redis
+
+For Redis, we will attach the [Heroku Redis addon][16] to our Heroku application:
+
+```shell
+heroku addons:create heroku-redis:hobby-dev
+```
+
+To check that Redis has been succesfully attached to your application, you can run the following command:
+
+ ```shell
+heroku addons:info REDIS
+```
+
+You should get an output similar to the following:
+
+```shell
+=== redis-cylindrical-59589
+Attachments:  ruby-heroku-datadog::REDIS
+Installed at: Wed Nov 17 2021 14:14:13 GMT+0100 (Central European Standard Time)
+Owning app:   ruby-heroku-datadog
+Plan:         heroku-redis:hobby-dev
+Price:        free
+State:        created
+```
+
+And retrieve the credentials from HEROKU by running the following command:
+
+```shell
+heroku config -a $APPNAME | grep REDIS_URL
+```
+
+Create a configuration file called `/datadog/conf.d/redisdb.yaml` at the root of your application replacing your host, port and password with the information you got in the previous command:
+
+```yaml
+init_config:
+
+instances:
+  - host: <YOUR_REDIS_HOST>
+    password: <YOUR_REDIS_PASSWORD>
+    port: <YOUR_REDIS_PORT>
+```
+
+Instead of manually replacing and harcoding the configuration, you can also set up your Redis integration based on Heroku environment variables, using the [prerun script][14] to replace those values before starting the Datadog agent:
```

Comment:
```suggestion
Instead of manually updating the configuration, you can set up your Redis integration based on Heroku environment variables using the [prerun script][14] to replace those values before starting the Datadog Agent:
```